ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -521,5 +521,8 @@ open class BadgeView: UIView, TokenizedControlInternal {
 
     // MARK: Accessibility
 
-    open override var accessibilityLabel: String? { get { return label.text } set { } }
+    open override var accessibilityLabel: String? {
+        get { return dataSource?.accessibilityLabel ?? label.text }
+        set { dataSource?.accessibilityLabel = newValue }
```

Comment:
I'm not sure we need the set anymore. People should be able to update the accessibility label through the dataSource, so I don't think we need it exposed in two places.